src/check/canonicalize/Diagnostic.zig

```diff
@@ -947,15 +947,15 @@ pub const Diagnostic = union(enum) {
         var report = Report.init(gpa, "UNUSED VARIABLE", .warning);
         const owned_ident = try report.addOwnedString(ident_name);
 
-        try report.document.addText("Variable `");
+        try report.document.addText("Variable ");
         try report.document.addUnqualifiedSymbol(owned_ident);
-        try report.document.addText("` is not used anywhere in your code.");
+        try report.document.addText(" is not used anywhere in your code.");
         try report.document.addLineBreak();
         try report.document.addLineBreak();
 
         try report.document.addText("If you don't need this variable, prefix it with an underscore like `_");
         try report.document.addText(owned_ident);
-        try report.document.addText("` to suppress this warning.");
+        try report.document.addText(" to suppress this warning.");
```

Comment:
```suggestion
        try report.document.addText("` to suppress this warning.");
```

Reply:
I'm thinking we need to do a pass over the diagnostics at some point